packages/frontend/src/components/create-campaign/form/kpi-step/simulation-chart/index.tsx

```diff
@@ -0,0 +1,313 @@
+import {
+    Area,
+    CartesianGrid,
+    ComposedChart,
+    Label,
+    ReferenceLine,
+    ResponsiveContainer,
+    Tooltip,
+    XAxis,
+    YAxis,
+} from "recharts";
+import { useMemo, useState } from "react";
+import { useTranslations } from "next-intl";
+import { Typography } from "@metrom-xyz/ui";
+import { TvlTick } from "./axis-ticks/tvl";
+import { RewardTick } from "./axis-ticks/reward";
+import { TooltipContent, TooltipCursor } from "./tooltip";
+import {
+    getDistributableRewardsPercentage,
+    getReachedGoalPercentage,
+} from "@/src/utils/kpi";
+
+import styles from "./styles.module.css";
+import { useDebounce } from "react-use";
+
+export interface ChartData {
+    tvl: number;
+    reward: number;
+    goalReachedPercentage: number;
+}
+
+interface SimulationChartProps {
+    minimumPayoutPercentage?: number;
+    lowerUsdTarget?: number;
+    upperUsdTarget?: number;
+    totalRewardsUsd?: number;
+    poolUsdTvl?: number | null;
+}
+
+const POINTS_COUNT = 1000;
+
+export function SimulationChart({
+    minimumPayoutPercentage: minPayoutPercentage = 0,
+    lowerUsdTarget: lowerBound,
+    upperUsdTarget: upperBound,
+    totalRewardsUsd,
+    poolUsdTvl,
+}: SimulationChartProps) {
+    const t = useTranslations("simulationChart");
+
+    const [lowerUsdTarget, setlowerUsdTarget] = useState(lowerBound);
+    const [upperUsdTarget, setupperUsdTarget] = useState(upperBound);
```

Comment:
Missing `camelCase` on the `set` part of the `useState`

Reply:
Fixed in 0edb2c51d297f15fb54f2bfbba97283afc4c78e6.